### ISSUE TITLE
Android studio create project does not configure package correctly

### ISSUE THREAD

**Issue by branflake2267** [flutter/flutter-intellij#1674]
Android Studio creates the project incorrectly. 

## Steps to Reproduce
1. Create a flutter project from Android studio
2. Name the project, someproject
3. Name the package domain.tld, and notice that the package id would be tld.domain.someproject
4. Finish creating project
5. Open up the android package, go to the android manifest
6. Verify the package is now tld.domain.someproject.someproject, which is incorrect.
7. Verify the package is tld.domain.someproject.someproject

## Version info
```
brandons-mbp:economy branflake2267$ flutter doctor
[✓] Flutter (on Mac OS X 10.13.2 17C205, locale en-US, channel master)
    • Flutter version 0.0.21-pre.325 at /Users/branflake2267/git/flutter
    • Framework revision b2c65e7653 (35 hours ago), 2018-01-27 14:21:58 +0100
    • Engine revision 6921873c71
    • Tools Dart version 2.0.0-dev.16.0
    • Engine Dart version 2.0.0-edge.da1f52592ef73fe3afa485385cb995b9aec0181a

[✓] Android toolchain - develop for Android devices (Android SDK 27.0.2)
    • Android SDK at /Users/branflake2267/Library/Android/sdk
    • Android NDK at /Users/branflake2267/Library/Android/sdk/ndk-bundle
    • Platform android-27, build-tools 27.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-915-b08)

[✓] iOS toolchain - develop for iOS devices (Xcode 9.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.2, Build version 9C40b
    • ios-deploy 1.9.2
    • CocoaPods version 1.3.1

[✓] Android Studio (version 3.0)
    • Android Studio at /Applications/Android Studio 3.0.app/Contents
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-915-b08)

[✓] Android Studio (version 3.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-915-b08)

[✓] IntelliJ IDEA Ultimate Edition (version 2017.3.1)
    • Flutter plugin version 20.0.3
    • Dart plugin version 173.3942.31

[✓] IntelliJ IDEA Community Edition (version 2017.2.5)
    • Flutter plugin version 17.0
    • Dart plugin version 172.4155.35

[✓] Connected devices
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 8.0.0 (API 26) (emulator)

brandons-mbp:economy branflake2267$ 
```

**Comment by branflake2267** [comment]
It appears selecting a custom directory may play a part of this issue. It looks like the module name could be determined from the directory name possibly.

**Comment by stevemessick** [comment]
Fixed with #1748
